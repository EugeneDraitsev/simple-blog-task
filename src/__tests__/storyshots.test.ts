import initStoryShots from '@storybook/addon-storyshots'
import 'jest-styled-components'

initStoryShots({})
