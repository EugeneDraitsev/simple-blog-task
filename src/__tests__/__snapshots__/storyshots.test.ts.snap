// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Buttons Simple buttons 1`] = `
.c1 {
  cursor: pointer;
  display: inline-block;
  min-height: 1em;
  outline: 0;
  border: none;
  vertical-align: baseline;
  background: #E6E6E6;
  color: #828282;
  font-family: Roboto,sans-serif;
  padding: 20px;
  text-shadow: none;
  font-weight: 700;
  line-height: 1em;
  font-size: 14px;
  font-style: normal;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  box-shadow: 0 0 0 1px transparent inset,0 0 0 0 rgba(34,36,38,.15) inset;
  text-transform: uppercase;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: 0.3s background-color;
  transition: 0.3s background-color;
}

.c1:hover {
  background: #cdcdcd;
}

.c1[disabled] {
  background: #808080;
  color: #e8e8e8;
  cursor: auto;
}

.c3 {
  background: #ca5a5a;
  color: #FFFFFF;
}

.c3:hover {
  background: #b63b3b;
}

.c3[disabled] {
  background: #e3a7a7;
  cursor: auto;
}

.c2 {
  background: #349884;
  color: #FFFFFF;
}

.c2:hover {
  background: #277263;
}

.c2[disabled] {
  background: #67cbb7;
  cursor: auto;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
>
  <button
    className="c1"
  >
    Default
  </button>
  <button
    className="c2 c1"
  >
    Primary
  </button>
  <button
    className="c3 c1"
  >
    Danger
  </button>
  <button
    className="c1"
    disabled={true}
  >
    Disabled
  </button>
  <button
    className="c2 c1"
    disabled={true}
  >
    Disabled
  </button>
  <button
    className="c3 c1"
    disabled={true}
  >
    Disabled
  </button>
  <button
    className="c1"
  >
    üçî Emoji
  </button>
  <button
    className="c2 c1"
  >
    üçî Emoji
  </button>
</div>
`;

exports[`Storyshots Modal Modal with animation 1`] = `
.c4 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  width: 400px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 4px;
  padding: 20px;
  z-index: 4;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6 {
  font-size: 32px;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c8 {
  font-size: 40px;
  line-height: 40px;
  cursor: pointer;
  margin: 0 10px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
  z-index: 3;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-size: 40px;
  text-align: center;
  cursor: pointer;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c10 {
  padding: 0 5px;
  font-size: 16px;
}

.c3 {
  -webkit-transform: translate(-100vw,-50%) scale(0);
  -ms-transform: translate(-100vw,-50%) scale(0);
  transform: translate(-100vw,-50%) scale(0);
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3 c4"
    >
      <div
        className="c5"
      >
        <div
          className="c6"
        >
          test title
        </div>
        <div
          className="c7"
        />
        <div
          className="material-icons c8"
          onClick={[Function]}
        >
          close
        </div>
      </div>
      <div
        className="greeting"
      >
        <div
          className="c9"
        >
          <div
            className="c10"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Modal Modal with buttons 1`] = `
.c13 {
  cursor: pointer;
  display: inline-block;
  min-height: 1em;
  outline: 0;
  border: none;
  vertical-align: baseline;
  background: #E6E6E6;
  color: #828282;
  font-family: Roboto,sans-serif;
  padding: 20px;
  text-shadow: none;
  font-weight: 700;
  line-height: 1em;
  font-size: 14px;
  font-style: normal;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  box-shadow: 0 0 0 1px transparent inset,0 0 0 0 rgba(34,36,38,.15) inset;
  text-transform: uppercase;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: 0.3s background-color;
  transition: 0.3s background-color;
}

.c13:hover {
  background: #cdcdcd;
}

.c13[disabled] {
  background: #808080;
  color: #e8e8e8;
  cursor: auto;
}

.c14 {
  background: #ca5a5a;
  color: #FFFFFF;
}

.c14:hover {
  background: #b63b3b;
}

.c14[disabled] {
  background: #e3a7a7;
  cursor: auto;
}

.c4 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  width: 400px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 4px;
  padding: 20px;
  z-index: 4;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6 {
  font-size: 32px;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c8 {
  font-size: 40px;
  line-height: 40px;
  cursor: pointer;
  margin: 0 10px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
  z-index: 3;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c3 {
  -webkit-transform: translate(-100vw,-50%) scale(0);
  -ms-transform: translate(-100vw,-50%) scale(0);
  transform: translate(-100vw,-50%) scale(0);
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  justify-items: flex-end;
  width: 100%;
  margin-top: 20px;
}

.c11 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c12 {
  margin-right: 10px;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-size: 40px;
  text-align: center;
  cursor: pointer;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3 c4"
    >
      <div
        className="c5"
      >
        <div
          className="c6"
        >
          Are you sure?
        </div>
        <div
          className="c7"
        />
        <div
          className="material-icons c8"
          onClick={[Function]}
        >
          close
        </div>
      </div>
      <div
        className="greeting"
      >
        <div
          className="c9"
        >
          Are you really want to delete your post?
          <div
            className="c10"
          >
            <div
              className="c11"
            />
            <button
              className="c12 c13"
              onClick={[Function]}
            >
              Cancel
            </button>
            <button
              className="c14 c13"
              onClick={[Function]}
            >
              Delete
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Modal Modal with overlay 1`] = `
.c3 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  width: 400px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 4px;
  padding: 20px;
  z-index: 4;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c6 {
  font-size: 40px;
  line-height: 40px;
  cursor: pointer;
  margin: 0 10px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
  z-index: 3;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-size: 40px;
  text-align: center;
  cursor: pointer;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c8 {
  padding: 0 5px;
  font-size: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3"
    >
      <div
        className="c4"
      >
        <div
          className="c5"
        />
        <div
          className="material-icons c6"
          onClick={[Function]}
        >
          close
        </div>
      </div>
      <div
        className="greeting"
      >
        <div
          className="c7"
        >
          <div
            className="c8"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Modal Modal with title 1`] = `
.c3 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  width: 400px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 4px;
  padding: 20px;
  z-index: 4;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  font-size: 32px;
}

.c6 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c7 {
  font-size: 40px;
  line-height: 40px;
  cursor: pointer;
  margin: 0 10px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
  z-index: 3;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-size: 40px;
  text-align: center;
  cursor: pointer;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c9 {
  padding: 0 5px;
  font-size: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3"
    >
      <div
        className="c4"
      >
        <div
          className="c5"
        >
          test title
        </div>
        <div
          className="c6"
        />
        <div
          className="material-icons c7"
          onClick={[Function]}
        >
          close
        </div>
      </div>
      <div
        className="greeting"
      >
        <div
          className="c8"
        >
          <div
            className="c9"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Spinner Spinners üåÄ 1`] = `
.c1 {
  -webkit-animation: iVXCSc 2s linear infinite;
  animation: iVXCSc 2s linear infinite;
  height: 100%;
  -webkit-transform-origin: center center;
  -ms-transform-origin: center center;
  transform-origin: center center;
  width: 100%;
  margin: auto;
}

.c2 {
  -webkit-animation: jVqWEn 1.5s ease-in-out infinite;
  animation: jVqWEn 1.5s ease-in-out infinite;
  stroke-dasharray: 1,200;
  stroke-dashoffset: 0;
  stroke-linecap: round;
  stroke: #349884;
}

.c3 {
  -webkit-animation: jVqWEn 1.5s ease-in-out infinite;
  animation: jVqWEn 1.5s ease-in-out infinite;
  stroke-dasharray: 1,200;
  stroke-dashoffset: 0;
  stroke-linecap: round;
  stroke: red;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
>
  <svg
    className="c1"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="c2"
      color=""
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={5}
    />
  </svg>
  <svg
    className="c1"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="c2"
      color=""
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={1}
    />
  </svg>
  <svg
    className="c1"
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="c2"
      color=""
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={5}
    />
  </svg>
  <svg
    className="c1"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="c3"
      color="red"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={5}
    />
  </svg>
</div>
`;
